make failed

### What happens?

running `make` failed with the following log, and it seems to be related to [this](https://www.reddit.com/r/Ubuntu/comments/16gzqvq/ubuntuports_suddenly_missing_kinetic/).

```
command -v emcc &> /dev/null && EXEC_ENVIRONMENT="" && echo '\033[1m=== Using native mode ===\033[0m' && echo 'Emscripten from' && which emcc || (EXEC_ENVIRONMENT=echo docker compose run duckdb-wasm-ci && echo '\033[1m === Using docker environment === \033[0m')
[+] Building 0.0s (0/0)                                                                                                                           docker:desktop-linux
[+] Running 1/1
 ! duckdb-wasm-ci Warning                                                                                                                                         4.0s 
[+] Building 5.4s (5/11)                                                                                                                          docker:desktop-linux
 => [duckdb-wasm-ci internal] load .dockerignore                                                                                                                  0.0s
 => => transferring context: 2B                                                                                                                                   0.0s
 => [duckdb-wasm-ci internal] load build definition from Dockerfile                                                                                               0.0s
 => => transferring dockerfile: 3.45kB                                                                                                                            0.0s
 => [duckdb-wasm-ci internal] load metadata for docker.io/library/ubuntu:22.10                                                                                    2.8s
 => CACHED [duckdb-wasm-ci 1/8] FROM docker.io/library/ubuntu:22.10@sha256:e322f4808315c387868a9135beeb11435b5b83130a8599fd7d0014452c34f489                       0.0s
 => ERROR [duckdb-wasm-ci 2/8] RUN apt-get update -qq     && DEBIAN_FRONTEND=noninteractive apt-get install -y         tini build-essential git         ccache c  2.6s
------
 > [duckdb-wasm-ci 2/8] RUN apt-get update -qq     && DEBIAN_FRONTEND=noninteractive apt-get install -y         tini build-essential git         ccache cmake ninja-build llvm clang clang-format clang-tidy curl python3         bison flex         brotli rsync         libpthread-stubs0-dev         libboost-all-dev         chromium-browser         wget gnupg ca-certificates procps libxss1         zip sqlite3         software-properties-common     && add-apt-repository ppa:mozillateam/ppa     && mkdir -p /etc/apt/preferences.d/     && apt-get update -qq     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'     && apt-get update     && apt-get install -y google-chrome-stable     && apt-get clean     && rm -rf /var/lib/apt/lists/* /etc/apt/sources.list.d/google.list:
2.568 E: The repository 'http://ports.ubuntu.com/ubuntu-ports kinetic Release' does not have a Release file.
2.568 E: The repository 'http://ports.ubuntu.com/ubuntu-ports kinetic-updates Release' does not have a Release file.
2.568 E: The repository 'http://ports.ubuntu.com/ubuntu-ports kinetic-backports Release' does not have a Release file.
2.568 E: The repository 'http://ports.ubuntu.com/ubuntu-ports kinetic-security Release' does not have a Release file.
------
failed to solve: process "/bin/sh -c apt-get update -qq     && DEBIAN_FRONTEND=noninteractive apt-get install -y         tini build-essential git         ccache cmake ninja-build llvm clang clang-format clang-tidy curl python3         bison flex         brotli rsync         libpthread-stubs0-dev         libboost-all-dev         chromium-browser         wget gnupg ca-certificates procps libxss1         zip sqlite3         software-properties-common     && add-apt-repository ppa:mozillateam/ppa     && mkdir -p /etc/apt/preferences.d/     && apt-get update -qq     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -     && sh -c 'echo \"deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main\" >> /etc/apt/sources.list.d/google.list'     && apt-get update     && apt-get install -y google-chrome-stable     && apt-get clean     && rm -rf /var/lib/apt/lists/* /etc/apt/sources.list.d/google.list" did not complete successfully: exit code: 100
make: *** [set_environment] Error 17
```

### To Reproduce

run `make`

### Browser/Environment:

macos 14.0, m1 pro

### Device:

macbook pro

### DuckDB-Wasm Version:

1.28.0

### DuckDB-Wasm Deployment:

local

--- a/actions/image/Dockerfile
+++ b/actions/image/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:22.10
+FROM ubuntu:23.10
 
 RUN apt-get update -qq \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y \

